docs/claims.md

```diff
@@ -0,0 +1,175 @@
+---
+id: claims
+title: Polkadot Claims
+sidebar_label: Claims
+---
+
+When Polkadot enters into its soft launch, the functionality for making on-chain claims for DOTs
+will already be enabled. If you've already made a pre-genesis claim by making a claim transaction on
+the Ethereum Claim Contract, then you will not need to claim again but you will need to send a free
+_attest_ transaction before your DOTs are sent to your account.
+
+This guide will walk you through the steps for either making a claim or attesting to the statement.
+
+If you are making a claim on Polkadot for the first time please read on below in the
+[making a claim](#making-a-claim) section. If you've already claimed during the pre-genesis claims
+please proceed to [attesting to a statem](#attesting-to-a-statement) section instead.
+
+## Making a Claim
+
+If you did not make a claim in the pre-genesis claims period then you are able to make a claim any
+time after genesis. There is no time limit for making your claim so feel free to do this whenever
+you are most comfortable.
+
+> Note: When you make a claim, at the same time you will make an attestation to the required
+> statement. The two actions "claim" and "attest" are done at once in a `claim_attest` transaction,
+> but for the most part this is simplified from the perspective of the UI.
+
+### What you will need
+
+- The Ethereum account that holds the DOT indicator tokens.
+- The MyCrypto wallet.
+- A Polkadot account.
+
+You should already have your Ethereum account that holds the DOT indicator tokens from Polkadot's
+prior sales. You will need to have access to this account in order to make a signature.
+
+[MyCrypto][] is a versatile wallet that supports a variety of storage methods for your Ethereum
+account. Please click on the link to go to their downloads page and ensure you download the latest
+version for your operating system. This is important because the latest version will always have the
+latest security patches.
+
+You will need a Polkadot account to mark as the destination for sending the DOTs in the claim.
+Please follow the instructions on the [account generation][] page for generating a new Polakdot
+account.
+
+#### Claiming your DOT with MyCrypto
+
+The Polkadot JS [Claims app][] helps you sign a message from MyCrypto. MyCrypto is a good choice in
+case you have stored the key to the Ethereum account holding your DOT indicator tokens on a hardware
+device like a Ledger Nano S or a Trezor. It also supports raw private keys, mnemonics and Parity
+signer.
+
+**NOTICE**: It is much more secure to download and use the MyCrypto app locally. Please make sure to
```

Comment:
you just said this a few paragraphs ago

Reply:
We really want to get the point across :) 

.. but point taken, will remove one of them.